engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/EngineJobExecutionActor.scala

```diff
@@ -146,7 +167,11 @@ class EngineJobExecutionActor(replyTo: ActorRef,
       saveJobCompletionToJobStore(data.withSuccessResponse(response))
     case Event(response: BackendJobExecutionResponse, data: ResponsePendingData) =>
       // This matches all response types other than `SucceededResponse`.
-      log.error("{}: Failed copying cache results, falling back to running job.", jobDescriptorKey)
+      response match {
+        case f: FailedNonRetryableResponse =>log.error("{}: Failed copying cache results, falling back to running job: {}", jobDescriptorKey, f.throwable)
+        case f: FailedRetryableResponse =>log.error("{}: Failed copying cache results, falling back to running job: {}", jobDescriptorKey, f.throwable)
+        case _ => //
```

Comment:
It could also be an AbortedResponse (if/when we wire that through to the cache hit actor...). It also stops scala from whining about incomplete case statements... :)

But I'll definitely merge those failed two cases. That's just lazy copy/pasting...